casper/src/test/scala/coop/rchain/casper/batch1/MultiParentCasperCommunicationSpec.scala

```diff
@@ -88,88 +90,88 @@ class MultiParentCasperCommunicationSpec extends FlatSpec with Matchers with Ins
+//      for {
+//        _ <- makeDeploy(0) >>= (nodes(0).addBlock(_))
+//
+//        _ <- nodes(1).syncWith(nodes(0))
+//        _ <- nodes(2).shutoff() //nodes(2) misses this block
+//      } yield ()
+//
+//    TestNode.networkEff(genesis, networkSize = 3).use { nodes =>
+//      for {
+//        _ <- stepSplit(nodes) // blocks a1 a2
+//        _ <- stepSplit(nodes) // blocks b1 b2
+//        _ <- stepSplit(nodes) // blocks c1 c2
+//
+//        _ <- stepSingle(nodes) // block d1
+//        _ <- stepSingle(nodes) // block e1
+//
+//        _ <- stepSplit(nodes) // blocks f1 f2
+//        _ <- stepSplit(nodes) // blocks g1 g2
+//
+//        // this block will be propagated to all nodes and force nodes(2) to ask for missing blocks.
+//        br <- makeDeploy(0) >>= (nodes(0).addBlock(_)) // block h1
+//
+//        _ <- TestNode.propagate(nodes) // force the network to communicate
+//
+//        _ <- nodes(2).contains(br.blockHash) shouldBeF true
+//
+//        nr <- makeDeploy(0) >>= (nodes(2).addBlock(_))
+//      } yield { nr.header.parentsHashList shouldBe List(br.blockHash) }
+//    }
+//  }
+
+//  //TODO: investigate this test - it doesnt make much sense in the presence of hashes (see RCHAIN-3819)
+//  // and why on earth does it test logs?
+//  it should "handle a long chain of block requests appropriately" ignore effectTest {
+//    TestNode
+//      .networkEff(genesis, networkSize = 2)
+//      .use { nodes =>
+//        for {
+//          _ <- (0 to 9).toList.traverse_[Effect, Unit] { i =>
+//                for {
+//                  deploy <- ConstructDeploy.basicDeployData[Effect](i)
+//                  block  <- nodes(0).addBlock(deploy)
+//                  _      <- nodes(1).shutoff() //nodes(1) misses this block
+//                } yield ()
+//              }
+//          deployData10 <- ConstructDeploy.basicDeployData[Effect](10)
+//          block11      <- nodes(0).addBlock(deployData10)
+//
+//          // Cycle of requesting and passing blocks until block #3 from nodes(0) to nodes(1)
+//          _ <- (0 to 8).toList.traverse_[Effect, Unit] { i =>
+//                nodes(1).receive() >> nodes(0).receive() >> nodes(1).receive() >> nodes(0)
+//                  .receive() >> nodes(1).receive() >> nodes(0).receive()
+//              }
+//
+//          // We simulate a network failure here by not allowing block #2 to get passed to nodes(1)
+//
+//          // And then we assume fetchDependencies eventually gets called
+//          _ <- nodes(1).casperEff.fetchDependencies
+//          _ <- nodes(0).receive()
+//          _ <- nodes(0).receive()
+//
+//          _ = nodes(1).logEff.infos.count(_ startsWith "Requested missing block") should be(10)
+//          result = nodes(0).logEff.infos.count(
+//            s => (s startsWith "Received request for block") && (s endsWith "Response sent.")
+//          ) should be(10)
+//          // Oh yes - 10!
+//        } yield result
+//      }
+//  }
```

Comment:
We need note/issue to track removal/refactor of these tests (maybe this is already covered in some issue).

Reply:
https://github.com/rchain/rchain/issues/3277#issuecomment-798109618